tests/rt.conf

```diff
@@ -148,26 +148,26 @@ RUN     | control_wam_debug
-
-COMPILE | -DAPP=HAFS-ALL -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_gfdlmp_tedmf_nonsst -D32BIT=ON                        | - wcoss_cray wcoss2.intel | fv3 |
-RUN     | hafs_regional_docn                                                                                                      | - wcoss_cray wcoss2.intel               | fv3 |
-RUN     | hafs_regional_docn_oisst                                                                                                | - wcoss_cray wcoss2.intel               | fv3 |
-RUN     | hafs_regional_datm_cdeps                                                                                                | - wcoss_cray wcoss2.intel               | fv3 |
+COMPILE | -DAPP=HAFSW -DMOVING_NEST=ON -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_gfdlmp_tedmf_nonsst,FV3_HAFS_v0_thompson_tedmf_gfdlsf -D32BIT=ON |                | fv3 |
+RUN     | hafs_regional_atm                                                                                                       |                                         | fv3 |
+RUN     | hafs_regional_atm_thompson_gfdlsf                                                                                       |                                         | fv3 |
+RUN     | hafs_regional_atm_ocn                                                                                                   |                                         | fv3 |
+RUN     | hafs_regional_atm_wav                                                                                                   |                                         | fv3 |
+RUN     | hafs_regional_atm_ocn_wav                                                                                               |                                         | fv3 |
+RUN     | hafs_regional_1nest_atm                                                                                                 | - jet.intel                             | fv3 |
+RUN     | hafs_regional_telescopic_2nests_atm                                                                                     | - jet.intel                             | fv3 |
+RUN     | hafs_global_1nest_atm                                                                                                   | - jet.intel                             | fv3 |
+RUN     | hafs_global_multiple_4nests_atm                                                                                         | - jet.intel wcoss_cray                  | fv3 |
+RUN     | hafs_regional_specified_moving_1nest_atm                                                                                | - jet.intel                             | fv3 |
+RUN     | hafs_regional_storm_following_1nest_atm                                                                                 | - jet.intel hera.intel                  | fv3 |
```

Comment:
Please turn this test back on for hera.intel

Reply:
Done. Thanks, @DeniseWorthen!